src/pages/blog/index.en.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react'
 
 import Layout from '../../components/Layout'
-import BlogRoll from '../../components/BlogRoll'
+import BlogRoll from '../../components/BlogRoll_en'
```

Comment:
the BlogRoll component filter the posts with english language, in this way we exclude all the others.